main.nf

```diff
@@ -50,108 +56,16 @@ newgenome_fai = file("${params.newgenome}.fai")
 outfile_basename = file(params.outfile).getName()
 outfile_dir = file(params.outfile).getParent()
 
-/*
-* Index the new reference genome using bowtie_build
-*/
-if (!file("${newgenome_dir}/${newgenome_basename}.1.bt2").exists()){
-
-    process bowtieGenomeIndex {
```

Comment:
I must say, I'm a bit conflicted about dropping bowtie support entirely... I still think it would have made sense to support a suite of aligners and to be able to switch between them. However, I realise it would have required much more work in terms of postprocessing, and we don't have infinite bandwidth, so I understand why the decision was made

Reply:
I agree that keeping bowtie2 would be useful at least until we established that minimap2 is performing well (This is not clear yet). 
Eventually I would prefer settling on only one aligner to reduce the maintenance cost though

Reply:
Bowtie was reintroduced in a new workflow.